cves/kernel/CVE-2017-18017.yml

```diff
@@ -360,8 +370,9 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The person who originally wrote the code did not properly check memory was freed correctly.
```

Comment:
What exactly is not being freed correctly and possibly how should it be freed

Reply:
Clarified the issue and fix for memory issues.